src/app/app.routes.ts

```diff
@@ -0,0 +1,6 @@
+import { Route } from '@angular/router';
+import { AppComponent } from './app.component';
+
+export const appRoutes: Route[] = [
+  { path: '', component: AppComponent, pathMatch: 'full' },
```

Comment:
issue: we don't need this

Reply:
Removed! I wasn't aware this works by default.